webpack/assets/javascripts/foreman_hostgroups.js

```diff
@@ -1,10 +1,11 @@
 import $ from 'jquery';
+import { t_ } from './foreman_i18n';
```

Comment:
When i look at this file, does it make sense to call it `translate`?
```jsx
import { translate } from './foreman_i18n';

const warningMessage = translate('Some Puppet Classes are unavailable in the selected environment');
```

Some consumers can still use it as
```jsx
import { translate as __ } from './foreman_i18n';
// or
import { translate as t_ } from './foreman_i18n';
```